templates/src/graphql/schema.js

```diff
@@ -0,0 +1,51 @@
+const { gql } = require('apollo-server-express');
+
+const typeDefs = gql`

```

Comment:
Can you please add some documentation to the schema? https://www.apollographql.com/docs/apollo-server/schema/schema/#documentation-strings

Reply:
Added documentation to the schema. Can you please proofread them? Thanks.